### PR TITLE
Add test for HideFromTests

### PR DESCRIPTION
**Pull request by rehandalal**
Following up on #486 

r?

### CODE REVIEW

File: content/tests/components/common/HideFromTests.test.tsx [hunk 1]
```diff
@@ -3,12 +3,32 @@ import React from "react";
...
+}
+
 describe("HideFromTests", () => {
   afterEach(async () => {
+    global.__TESTING__ = true;
     await cleanup();
   });
 
-  it("should hide it's children", async () => {
+  it("show not hide it's children when testing", () => {
```
Comment (mythmon) [inline content/tests/components/common/HideFromTests.test.tsx]: Should this be "should not hide its children when not testing"?

**Review by mythmon — APPROVED** [review]
bors d+

File: content/tests/components/common/HideFromTests.test.tsx (hunk 1 above)
Comment (rehandalal) [reply content/tests/components/common/HideFromTests.test.tsx]: gah!

**Comment by rehandalal** [comment]
bors r=@mythmon

**Pull request merged**